src/test/java/edu/cornell/kfs/tax/batch/xml/TaxOutputDefinitionV2FileTypeTest.java

```diff
@@ -0,0 +1,226 @@
+                                type = TaxOutputFieldType.STATIC, value = "0.00"
+                        )
+                })
+        })
+        SINGLE_SECTION_MULTIPLE_FIELDS,
+
+        @TaxOutputDefinition(fieldSeparator = KFSConstants.COMMA, sections = {
+                @TaxOutputSection(name = "Biographic_Section", hasHeaderRow = true, fields = {
+                        @TaxOutputField(
+                                name = "recipient_id", length = 25,
+                                type = TaxOutputFieldType.DERIVED, key = "recipientId"
+                        ),
+                        @TaxOutputField(
+                                name = "full_name", length = 90,
+                                type = TaxOutputFieldType.DERIVED, key = "recipientName"
+                        ),
+                        @TaxOutputField(
+                                name = "primary_address", length = 120,
+                                type = TaxOutputFieldType.DERIVED, key = "recipientAddress"
+                        ),
+                        @TaxOutputField(
+                                name = "extra_address", length = 120,
+                                type = TaxOutputFieldType.STATIC, value = ""
+                        )
+                }),
+                @TaxOutputSection(name = "Payment_Section", hasHeaderRow = true, fields = {
+                        @TaxOutputField(
+                                name = "recipient_id", length = 25,
+                                type = TaxOutputFieldType.DERIVED, key = "recipientId"
+                        ),
+                        @TaxOutputField(
+                                name = "payment_source", length = 10,
+                                type = TaxOutputFieldType.STATIC, value = "Cornell"
+                        ),
+                        @TaxOutputField(
+                                name = "tax_amount", length = 15,
+                                type = TaxOutputFieldType.DERIVED, key = "taxAmount"
+                        )
+                })
+        })
+        MULTIPLE_SECTIONS_MULTIPLE_FIELDS;
+
+        public Named<TaxOutputDefinition> toNamedAnnotationFixturePayload() {
+            return FixtureUtils.createNamedAnnotationFixturePayload(this, TaxOutputDefinition.class);
+        }
+    }
+
+    static Stream<Arguments> validOutputDefinitions() {
+        return Stream.of(
+                Pair.of("good-file-single-section-single-field.xml", LocalTestCase.SINGLE_SECTION_SINGLE_FIELD),
+                Pair.of("good-file-single-section-multiple-fields.xml", LocalTestCase.SINGLE_SECTION_MULTIPLE_FIELDS),
+                Pair.of("good-file-multiple-sections-multiple-fields.xml",
+                        LocalTestCase.MULTIPLE_SECTIONS_MULTIPLE_FIELDS)
+        )
+                .map(pair -> Arguments.of(pair.getLeft(), pair.getRight().toNamedAnnotationFixturePayload()));
+    }
+
+
+
+    @ParameterizedTest
```

Comment:
Are the extra newlines by convention. I am fine with them if it helps separate the code visually

Reply:
I was trying to add some extra visual separation in certain classes (especially the unit test classes) without resorting to comments. Not sure if we have a specific convention, but I wanted some kind of extra separation to help make it easier for other developers to read the unit test code.